aiohttp_sse/__init__.py

```diff
@@ -25,7 +29,7 @@ def hello(request):
 
     DEFAULT_PING_INTERVAL = 15
 
-    def __init__(self, *, status=200, reason=None, headers=None):
+    def __init__(self, *, status=200, reason=None, headers=None, request=None):
```

Comment:
Or may be we should not inherit `EventSourceResponse` from `StreamingResponse` in the first place? Name class `EventSource`, then we free to pass parameters as we wish, and provide factory method to return preconfigured `StreamingResponse` instance.